packages/scripts/src/adapter-dependencies/.gitignore

```diff
@@ -0,0 +1 @@
+dependencies.txt
```

Comment:
is this intentional?

Reply:
Yes, this is an autogenerated file that we don't want to commit